pyspark/bigdl/keras/backend.py

```diff
@@ -109,15 +112,15 @@ def fit(self, x, y=None, batch_size=32, nb_epoch=10, verbose=1, callbacks=None,
         if validation_split != 0.:
             unsupport_exp("validation_split")
         bopt = self.__create_optimizer(x=x,
-                                y=y,
-                                batch_size=batch_size,
-                                nb_epoch=nb_epoch,
-                                validation_data=validation_data,
-                                is_distributed=is_distributed)
+                                       y=y,
```

Comment:
not sure what's the difference here? the code on server using "space" but you are using "tab" ?

Reply:
Emmm... just keep the parameters aligned... for the sake of better visualization... doesn't change the code anyway.